doc/vscode.md

```diff
@@ -30,31 +30,78 @@ It is still under development, thus might be incomplete and buggy.
 
 ##  Proof navigation
 
+
+
 Goals are visualised in a panel on the right side of the editor.
 
 - You can navigate the proof with [keybindings](#keybindings).
-- 
-- "Cursor mode" can be toggled to navigate the proff with the cursor.
+- "Cursor mode" can be toggled to navigate the proof with the cursor.
 
 ##  Snippets
 
-Typing "@letter" will suggest a corresponding unicode greek letter (for instance "@b" will suggest β). Press Enter or Tab to confirm adding the Unicode character.
+Type the advised snippets described below, then press Enter or Tab to confirm adding the chosen Unicode character.
 
-# Keybindings
+---
 
-- Navigate down : go forward in the proof
+###   Greek letters
 
-```
-Ctrl+Down
-```
+Typing ```@letter``` will suggest a corresponding unicode greek letter (for instance "@b" will suggest β). 
 
-- Navigate up : go backwards in the proof
+---
 
-```
-Ctrl+Up
-```
+###   Fonts
 
-- Toggle cursor mode
-```
-Ctrl+Alt+C
-```
\ No newline at end of file
+- ```@letterletter```(for instance @nn): double-struck letter (ℕ)
+
+- ```@itletter```(@itn): italic letter (𝑁)
+
+- ```@ibletter```(@bin): italic bold (𝑵)
+
+- ```@scletter```(@scn): script letter (𝒩 )
+
+- ```@bsletter```(@bsn): bold script letter (𝓝)
+
+- ```@frletter```(@frn): Fraktur letter (𝔑)
+
```

Comment:
I propose to simplify some prefixes: d for double-struck, i for italic, s for script, bs for bold script, f for fraktur. Isn't there a bold (without italic) too?